adapters/repos/db/vector/flat/index.go

```diff
@@ -695,7 +718,9 @@ func (index *flat) SwitchCommitLogs(context.Context) error {
 }
 
 func (index *flat) ListFiles(ctx context.Context, basePath string) ([]string, error) {
```

Comment:
is this taking care of backups? That the file is included?

Reply:
My understanding this is for backups. @parkerduckworth is this sufficient for backups to include the metadata file?

Reply:
@dirkkul I pushed a new commit that correctly handles `ListFiles` for backups and successfully validated that delete / restore works.